encord/project_ontology/ontology_object.py

```diff
@@ -1,3 +1,15 @@
+"""
+---
```

Comment:
Files in this folder are used, but long depreated. So it would be ideal to put them to some "deprecated" section or smth

Reply:
We can do that in the structure in Readme/Mintlify